src/App.js

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import ListProducts from './components/List_Products';
```

Comment:
why is the import still showing `List_Products`? 🤔

Reply:
Another thing is that it might be good to leave an empty line above this import to separate what comes from 3rd party libraries and the src folder itself

Reply:
_Interesting idea 👍_ 

```jsx
import React from 'react';

import ListProducts from './components/List_Products';
import './App.css';
```

Reply:
[I had to change back the file name because the deployment Action didn't recognize it](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/pull/16#discussion_r785431391).

Reply:
I arranged the imports as suggested.